### PR TITLE
Added Java and Kotlin code for AR customization

### PR DESCRIPTION
**Pull request by kiryldz**

### CODE REVIEW

File: Examples/src/main/kotlin/com/mapbox/vision/examples/ArCustomizationActivityKt.kt [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+    override fun setArRenderOptions(visionArView: VisionArView) {
+        // Make sure lane is visible
+        visionArView.setLaneVisible(true)
+        // Set the desired lane length in meters.
+        VisionArManager.setLaneLength(40f)
+
+        // Create an instance of LaneVisualParams
+        val laneVisualParams = LaneVisualParams(
+            // Set the color
```
Comment (RingerJK) [inline Examples/src/main/kotlin/com/mapbox/vision/examples/ArCustomizationActivityKt.kt]: imo `javadocs` for params of  `LaneVisualParams ` should be enough, don't need one more comment

File: Examples/src/main/kotlin/com/mapbox/vision/examples/ArCustomizationActivityKt.kt [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            arrowLength = 2.5f
+        )
+        // After `laneVisualParams` is configured you need to set it to `VisionArView`
+        visionArView.setLaneVisualParams(laneVisualParams)
+
+        // Enable fence rendering
+        visionArView.setFenceVisible(true)
+
+        // Create an instance of `FenceVisualParams`
+        val fenceVisualParams = FenceVisualParams(
```
Comment (RingerJK) [inline Examples/src/main/kotlin/com/mapbox/vision/examples/ArCustomizationActivityKt.kt]: the same as above

File: Examples/src/main/java/com/mapbox/vision/examples/ArCustomizationActivity.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+                3f,
+                // Set the number of arrows in the fence
+                3
+        );
+        // After `fenceVisualParams` is configured you need to set it to to `VisionArView`
+        visionArView.setFenceVisualParams(fenceVisualParams);
+        // Setting AR render overall quality a bit lower to gain performance
+        visionArView.setArQuality(0.8f);
+    }
+}
```
Comment (yuryybk) [inline Examples/src/main/java/com/mapbox/vision/examples/ArCustomizationActivity.java]: What if add this code to ARActivity? Looks simple enough for understanding and IMHO doesn't deserve separate activity
I'd extracted lane customizaiton and fence customization to separate methods too

File: Examples/src/main/kotlin/com/mapbox/vision/examples/ArCustomizationActivityKt.kt (hunk 1 above)
Comment (kiryldz) [reply Examples/src/main/kotlin/com/mapbox/vision/examples/ArCustomizationActivityKt.kt]: I've actually copied iOS code. So will leave just to be consistent. Moreover javadocs are not available via `aar`-s.

File: Examples/src/main/kotlin/com/mapbox/vision/examples/ArCustomizationActivityKt.kt (hunk 2 above)
Comment (kiryldz) [reply Examples/src/main/kotlin/com/mapbox/vision/examples/ArCustomizationActivityKt.kt]: the same as answer above 👍

File: Examples/src/main/java/com/mapbox/vision/examples/ArCustomizationActivity.java (hunk 3 above)
Comment (kiryldz) [reply Examples/src/main/java/com/mapbox/vision/examples/ArCustomizationActivity.java]: Well, AR quality is part of AR customization talking strictly. So I'd leave it in activity showing capabilities of our AR customization.

File: Examples/src/main/java/com/mapbox/vision/examples/ArCustomizationActivity.java [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+public class ArCustomizationActivity extends ArActivity {
+
+    @Override
+    protected void setArRenderOptions(@NonNull final VisionArView visionArView) {
+        // Make sure lane is visible
+        visionArView.setLaneVisible(true);
+        // Set the desired lane length in meters.
+        VisionArManager.setLaneLength(40f);
+
+        // Create an instance of LaneVisualParams
```
Comment (yunikkk) [inline Examples/src/main/java/com/mapbox/vision/examples/ArCustomizationActivity.java]: I'd remove this comment - it doesn't describe our API

File: Examples/src/main/java/com/mapbox/vision/examples/ArCustomizationActivity.java [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+        final LaneVisualParams laneVisualParams = new LaneVisualParams(
+                // Set the color
```
Comment (yunikkk) [inline Examples/src/main/java/com/mapbox/vision/examples/ArCustomizationActivity.java]: nit : `set red color` ?

File: Examples/src/main/java/com/mapbox/vision/examples/ArCustomizationActivity.java [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+                new Color(1.0f, 0.0f, 0.0f, 1.0f),
+                // Set the width
```
Comment (yunikkk) [inline Examples/src/main/java/com/mapbox/vision/examples/ArCustomizationActivity.java]: nit : `set width 1 meter` ?

File: Examples/src/main/java/com/mapbox/vision/examples/ArCustomizationActivity.java (hunk 3 above)
Comment (yunikkk) [reply Examples/src/main/java/com/mapbox/vision/examples/ArCustomizationActivity.java]: Agree with @kiryldz

File: Examples/src/main/java/com/mapbox/vision/examples/ArCustomizationActivity.java (hunk 5 above)
Comment (kiryldz) [reply Examples/src/main/java/com/mapbox/vision/examples/ArCustomizationActivity.java]: OK

File: Examples/src/main/java/com/mapbox/vision/examples/ArCustomizationActivity.java (hunk 4 above)
Comment (kiryldz) [reply Examples/src/main/java/com/mapbox/vision/examples/ArCustomizationActivity.java]: OK

File: Examples/src/main/java/com/mapbox/vision/examples/ArCustomizationActivity.java (hunk 6 above)
Comment (kiryldz) [reply Examples/src/main/java/com/mapbox/vision/examples/ArCustomizationActivity.java]: OK

File: Examples/src/main/java/com/mapbox/vision/examples/ArCustomizationActivity.java (hunk 3 above)
Comment (yuryybk) [reply Examples/src/main/java/com/mapbox/vision/examples/ArCustomizationActivity.java]: I mean not only quality but the whole code in method setArRenderOptions(). Sorry for bad code selection

**Pull request merged**